### PR TITLE
fix: update nusnet id initialization to uppercase

### PR DESCRIPTION
**Pull request by ForAeons**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/NusNet.java
Comment (saiutkarsh33) [inline src/main/java/seedu/address/model/person/NusNet.java:1]: Edit docs for constructor

**Review by saiutkarsh33 — CHANGES_REQUESTED** [review]
LGTM

**Review by saiutkarsh33 — APPROVED** [review]
LGTM

**Review by tituschewxj — APPROVED** [review]
LGTM!

**Pull request merged**